projects/visual_bert/configs/mmimdb/defaults.yaml

```diff
@@ -1,3 +1,6 @@
+includes:
+- configs/datasets/mmimdb/with_features.yaml
```

Comment:
Nice!! That you already did this.

Reply:
vilbert will also need similar change for mmimdb.

Reply:
I added the same lines for the vilbert case and just did PR.